src/zap/zapimage.cpp

```diff
@@ -1547,7 +1547,7 @@ void ZapImage::OutputTables()
 
 #if defined(FEATURE_PAL)
     // PAL library requires native image sections to align to page bounaries.
-    SetFileAlignment(0x1000);
+    SetFileAlignment(0x10000);
```

Comment:
@sdmaclea what do you mean by  RELOC_PAGE_SIZE? I don't see such a symbol defined anywhere.